extensions/npm/src/npmView.ts

```diff
@@ -276,7 +279,15 @@ export class NpmScriptsTreeDataProvider implements TreeDataProvider<TreeItem> {
 		return fullName === task.name;
 	}
 
-	private buildTaskTree(tasks: TaskWithLocation[]): Folder[] | PackageJSON[] | NoScripts[] {
+	private sortTaskTree(taskTree: TaskTree) {
+		return taskTree.sort((first, second) => {
+			const firstLabel = first.label?.toString() || '';
+			const secondLabel = second.label?.toString() || '';
+			return firstLabel.localeCompare(secondLabel);
```

Comment:
In the above test case, No second item to compare with.

Reply:
The second item will always be provided to the comparator function.